app/database_etl/owid_ingestion_handler/owid_data_handler.py

```diff
@@ -90,3 +91,8 @@ def get_vaccinated(country_name: str, midpoint_date: datetime, fully_vaccinated=
     col_name = 'people_fully_vaccinated_per_hundred' if fully_vaccinated else 'people_vaccinated_per_hundred'
     ret = country_data.loc[country_data['date'] == target_date][col_name]
     return float(ret) if not ret.empty else None
+
+
+def get_whether_exact_match(country_name: str, estimate_grade: str):
```

Comment:
Suggestion: we could make this function have 2 arguments `record_estimate_grade` and `owid_data_estimate_grade` (probably needs a more appropriate name b/c owid won't always be our source but u get what I mean).

Then the logic would look like `return record_estimate_grade == owid_data_estimate_grade`. Functionally this would act the same way as our current logic. However imo this will make us less likely to forget to update this logic in the future when we do add more granular data. Just a suggestion though, not mandatory.

Reply:
Resolving for now – we will need to have a conversation around this mechanism